Gems/PhysX/Code/Source/EditorMeshColliderComponent.cpp

```diff
@@ -845,6 +869,11 @@ namespace PhysX
 
     void EditorMeshColliderComponent::UpdateShapeConfigurationScale()
     {
+        const AZ::Vector3& assetScale = m_proxyShapeConfiguration.m_physicsAsset.m_configuration.m_assetScale;
+        const bool isAssetScaleUniform =
+            AZ::IsClose(assetScale.GetX(), assetScale.GetY()) && AZ::IsClose(assetScale.GetX(), assetScale.GetZ());
```

Comment:
nit: As this is the same 'IsUniform' check mentioned above, maybe it is worth moving to a helper (or using an existing one if one exists - tagging @greerdv who may know this).

Reply:
Covered in the previous comment.